streaming/src/main/scala/org/apache/spark/streaming/receiver/ReceivedBlockHandler.scala

```diff
@@ -79,7 +93,7 @@ private[streaming] class BlockManagerBasedBlockHandler(
       throw new SparkException(
         s"Could not store $blockId to block manager with storage level $storageLevel")
     }
-    BlockManagerBasedStoreResult(blockId)
+    BlockManagerBasedStoreResult(blockId, numRecords)
```

Comment:
Both are wrong but counting it as zero means we end up suggesting no data was received. I personally think they should be counted as 1, we can doc it